acceptancetests/assess_caas_bootstrap.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+""" Test caas k8s cluster bootstrap
+"""
+
+from __future__ import print_function
+
+import argparse
+import logging
+import sys
+import os
+
+from deploy_stack import (
+    BootstrapManager,
+)
+from utility import (
+    add_basic_testing_arguments,
+    configure_logging,
+)
+
+from jujucharm import (
+    local_charm_path
+)
+
+
+__metaclass__ = type
+
+
+log = logging.getLogger("assess_caas_bootstrap")
+
+
+def assess_caas_bootstrap(client):
+    # Deploy k8s bundle to spin up k8s cluster
+    repository_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'repository')
```

Comment:
The local repository dir should be provided by the env var 'JUJU_REPOSITORY' (you'll see local_charm_path looks for it if repository is not passed.)

This seems a little opaque, but it does separate the location of the repo from the test into the testing environment (i.e. the CI test jobs define JUJU_REPOSITORY).

Reply:
you are right, we should use the `JUJU_REPOSITORY` rather than finding it from relative path.